src/sentry/web/frontend/debug/mail.py

```diff
@@ -189,40 +189,59 @@ def make_error_event(request, project, platform):
 
 
 def make_performance_event(project, sample_name: str):
-    with override_options(
-        {
-            "performance.issues.all.problem-detection": 1.0,
-            "performance.issues.n_plus_one_db.problem-creation": 1.0,
-            "performance.issues.n_plus_one_api_calls.problem-creation": 1.0,
-            "performance.issues.render_blocking_assets.problem-creation": 1.0,
-        }
-    ), Feature(
-        {
-            "organizations:performance-n-plus-one-api-calls-detector": True,
-            "organizations:performance-issues-render-blocking-assets-detector": True,
-        }
-    ):
-        timestamp = datetime(2017, 9, 6, 0, 0)
-        start_timestamp = timestamp - timedelta(seconds=3)
-
-        perf_data = dict(
-            load_data(
-                sample_name,
-                start_timestamp=start_timestamp,
-                timestamp=timestamp,
-            )
+    if options.get(
+        "performance.issues.create_issues_through_platform", True
+    ) and project.get_option("sentry:performance_issue_create_issue_through_platform", True):
+        # TODO check if features need to be overriden
+        event_id = "44f1419e73884cd2b45c79918f4b6dc4"
+        occurrence_data = SAMPLE_TO_OCCURRENCE_MAP[sample_name].to_dict()
+        occurrence_data["event_id"] = event_id
+        occurrence, group_info = process_event_and_issue_occurrence(
+            occurrence_data,
+            {
+                "event_id": event_id,
+                "project_id": project.id,
+                "timestamp": before_now(minutes=1).isoformat(),
+            },
```

Comment:
I've noticed date and time change on every load so we'll probably need to hardcode them like it's done below

Reply:
Probably yeah, I think this caused issues with flaky snapshot tests before, cc @ceorourke

Reply:
Yeah definitely hard code the timestamp like `timestamp = datetime(2017, 9, 6, 0, 0)` so you're not haunted by DMs telling your about flaky tests.